Makefile

```diff
@@ -0,0 +1,25 @@
+#
+# Makefile to run the business analytics software using metabase
+#
+
+#runs the docker-compose.yml showing logs
+up :
+	@echo "Starting Riff Business Analytics.";
```

Comment:
Why the trailing semicolon?

Reply:
Removed semi-colons. Was unclear about the syntax required.

Reply:
Don't stay unclear. If you don't understand the syntax, find and read the docs, don't guess. https://www.gnu.org/software/make/manual/